recipes/pydevd-pycharm/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pydevd-pycharm" %}
+{% set version = "222.3739.30" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydevd-pycharm-{{ version }}.tar.gz
+  sha256: 2503d6b95572f791ed0289bbf81b83f3dc170ea232fd14b40feffa9b57d19ba5
+
+build:
+  skip: true  # [win]
+  entry_points:
+    - pydevd = pydevd:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/libc.so.6
+    - /lib64/libstdc++.so.6
+    - $RPATH/libc.so.6
+    - $RPATH/libstdc++.so.6
```

Comment:
This should not be required for a python only package. It looks like the "source" tar has [pre-compiled binaries](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=548317&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=7244). This is not allowed. You must compile libraries with our compilers.